pkg/parse/events/builder.go

```diff
@@ -46,7 +46,7 @@ type PublishingGroupBuilder struct {
 func (t *PublishingGroupBuilder) Build() []Publisher {
 	var publishers []Publisher
 	if t.SyncPeriod > 0 {
-		publishers = append(publishers, NewResetOnRunAttemptPublisher(SyncEventType, t.Clock, t.SyncPeriod))
+		publishers = append(publishers, NewTimeDelayPublisher(SyncEventType, t.Clock, t.SyncPeriod))
```

Comment:
i think the doc.go needs updating to reflect these changes

Reply:
comment out of date, the `NewResetOnRunAttemptPublisher` is retained so doc.go doesn't need to be updated